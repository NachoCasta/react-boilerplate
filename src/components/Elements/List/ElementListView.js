import React from "react";
import { List } from "components/UI";
import Item from "./Item";

const ElementList = ({ elements, ...rest }) => (
    <List
        bordered
        itemLayout="vertical"
        dataSource={ elements }
        renderItem={id => <Item id={id} />}
        {...rest}
    />
);

export default ElementList;
