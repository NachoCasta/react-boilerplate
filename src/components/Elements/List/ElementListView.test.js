import React from "react";
import { shallow } from "enzyme";

import ElementListView from "./ElementListView";

describe("ElementListView", () => {
	let component;
	beforeEach(() => {
		component = shallow(<ElementListView />);
	});

	it("renders", () => {
		shallow(<ElementListView />);
	});

	it("matches snapshot", () => {
		expect(component).toMatchSnapshot();
	});
});
