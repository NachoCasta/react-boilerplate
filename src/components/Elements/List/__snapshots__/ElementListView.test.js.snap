// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ElementListView matches snapshot 1`] = `ShallowWrapper {}`;
